andela/travel_tool_front
# #160371362 Requester should be able to get and view notification

#### What does this PR do?
This PR ensures that a requester gets a notification when a new comment is posted

#### Description of Task to be completed?
- ensure that a requester is able to click on `view details`  to view details about comment

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-fetch-comments-notifications-160371362`
- Install dependencies and start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/50)
- Navigate to `localhost:5000` in a browser
- Post comment in the comment thread
- A Requester gets a notification

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160371362](https://www.pivotaltracker.com/story/show/160371362)

#### Screenshots (if appropriate)
<img width="1275" alt="screen shot 2018-09-18 at 9 43 51 pm" src="https://user-images.githubusercontent.com/26988297/45715365-021bec00-bb8c-11e8-9e23-d061af11b9f3.png">


#### Questions:
